_data/sites.json

```
@@ -2736,6 +2736,17 @@
         ]
     },
 
+    {
+        "name": "Chick-fil-A",
+        "url": "https://privacyportal.onetrust.com/webform/63dc78c7-5612-4181-beae-47dead0569ee/01252c81-82df-4db5-9cb8-bf23a62a9a55",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```

I think it seems hard, I doubt the form is automatically processed, what do you think?

---

Web forms like these (such as OneTrust, TrustArc, and others) are manually managed by a company's privacy team but some parts like email verification to confirm your identity are automated. Depending on the company, they may or may not contact you again to ask for more info - but Chick-fil-A does not from my experience which is why I personally would say **_medium._**

---

Ok, fair enough!